packages/browser/src/lib/client-hints/interfaces.ts

```diff
@@ -0,0 +1,43 @@
+// WICG Spec: https://wicg.github.io/ua-client-hints
+
+declare interface Navigator extends NavigatorUA {}
+declare interface WorkerNavigator extends NavigatorUA {}
+
+// https://wicg.github.io/ua-client-hints/#navigatorua
+declare interface NavigatorUA {
```

Comment:
nit: I am proposing that we don't do any "declare" stuff here, since it's acts a kind of side effect for any consumer (these interfaces are global). We should just export the interfaces into our code as regular types. Thoughts?

Reply:
Sounds good! I didn't know declare had that effect.